MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.NumericUpDown.xaml

```diff
@@ -0,0 +1,56 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+                    xmlns:po="http://schemas.microsoft.com/winfx/2006/xaml/presentation/options"
+                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+                    xmlns:converters="clr-namespace:MaterialDesignThemes.Wpf.Converters"
+                    xmlns:local="clr-namespace:MaterialDesignThemes.Wpf"
+                    xmlns:system="clr-namespace:System;assembly=mscorlib"
+                    xmlns:transitions="clr-namespace:MaterialDesignThemes.Wpf.Transitions"
+                    xmlns:wpf="clr-namespace:MaterialDesignThemes.Wpf">
+  <ResourceDictionary.MergedDictionaries>
+    <ResourceDictionary Source="MaterialDesignTheme.Button.xaml" />
+  </ResourceDictionary.MergedDictionaries>
+
+  <Style x:Key="MaterialDesignNumericUpDown" TargetType="{x:Type wpf:NumericUpDown}">
+    <Setter Property="MinWidth" Value="100"/>
+    <Setter Property="Padding" Value="4"/>
+    <Setter Property="HorizontalContentAlignment" Value="Center" />
+    <Setter Property="VerticalContentAlignment" Value="Center" />
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="{x:Type wpf:NumericUpDown}">
+          <Border Background="{TemplateBinding Background}"
+                  BorderBrush="{TemplateBinding BorderBrush}"
+                  BorderThickness="{TemplateBinding BorderThickness}"
+                  CornerRadius="10"
+                  ClipToBounds="True">
+            <Grid>
+              <Grid.ColumnDefinitions>
+                <ColumnDefinition />
+                <ColumnDefinition Width="Auto" />
+                <ColumnDefinition Width="Auto" />
+              </Grid.ColumnDefinitions>
+
+              <TextBox x:Name="PART_TextBoxField"
+                       Margin="4 0"
+                       VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
+                       HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}" />
+
+              <RepeatButton x:Name="PART_DecreaseButton"
+                            Content="{local:PackIcon Kind=Minus}"
+                            Style="{StaticResource MaterialDesignFlatButton}"
+                            Padding="8,4"
+                            Grid.Column="1"/>
+              <RepeatButton x:Name="PART_IncreaseButton"
+                            Content="{local:PackIcon Kind=Plus}"
+                            Style="{StaticResource MaterialDesignFlatButton}"
+                            Padding="8,4"
+                            Grid.Column="2" />
+            </Grid>
+          </Border>
+        </ControlTemplate>
+      </Setter.Value>
+    </Setter>
+  </Style>
+
+  <Style TargetType="wpf:NumericUpDown" BasedOn="{StaticResource MaterialDesignNumericUpDown}" />
```

Comment:
These default implicit styles for the custom controls are typically set up in `Generic.xaml`; see [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/7f27b729df9ed950e339a17d7aa0bce1bc46f3ce/MaterialDesignThemes.Wpf/Themes/Generic.xaml#L37-L44).